download_data.py

```diff
@@ -58,3 +59,20 @@ def download_data(
         dataset_name=args.dataset_name,
         dataset_subset=args.dataset_subset,
         dataset_root_dir=args.dataset_dir)
+
+    # Initialize, setup, and parse the argument parser
+    parser = ArgumentParser(prog="Data Downloader")
+
+    parser.add_argument("--dataset-name", type=str, required=True,
+        help="Hugging Face dataset name. Should also set --dataset-subset.")
+    parser.add_argument("--dataset-subset", type=str, required=True,
+        help="Subset/config to use for Hugging Face dataset.")
+    parser.add_argument("--dataset-dir", type=str, required=True,
```

Comment:
Please order alphabetically.

Reply:
Addressed in [8fc2164](https://github.com/DRAGNLabs/301r_retnet/pull/25/commits/8fc2164d7bc3e273e84395655a0c87c648342f46).